ionoscloud/data_source_template_test.go

```diff
@@ -22,37 +22,31 @@ func TestAccDataSourceTemplate(t *testing.T) {
 			{
 				Config: testAccDataSourceTemplateName,
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(templateName, "name", "CUBES S"),
-					resource.TestCheckResourceAttr(templateName, "cores", "1"),
-					resource.TestCheckResourceAttr(templateName, "ram", "2048"),
-					resource.TestCheckResourceAttr(templateName, "storage_size", "50"),
+					resource.TestCheckResourceAttr(templateName, "name", "Basic Cube S"),
+					resource.TestCheckResourceAttr(templateName, "cores", "2"),
+					resource.TestCheckResourceAttr(templateName, "ram", "4096"),
+					resource.TestCheckResourceAttr(templateName, "storage_size", "120"),
 				),
 			},
 			{
-				Config: testAccDataSourceTemplateCores,
+				Config: testAccDataSourceTemplateCoresRam,
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(templateName, "name", "CUBES XL"),
```

Comment:
@cristiGuranIonos why are we removing these checks?

Reply:
easier to keep the test up-to-date when they change. they're tested in the first step, so they definitely work.